dali/base/dafdesc.cpp

```diff
@@ -1351,6 +1355,7 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
             if (!tree)
                 tree = createPTree("Attr");
             attr.setown(tree);
+            history.setown(createPTree("History"));
```

Comment:
Not sure needed here?
But would be better if didn't create/need a History node, as most common case is there will be no history (also see comment re. legacy/deserialization, need to handle files with none anyway)

Attr is a different case, it is rarely if ever empty.